<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="borrowPage">
      <attribute name="title">
       <string>借阅</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QLabel" name="borrowFileLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>借阅文件</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="borrowFileTitle">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="outputFileExists">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>不存在</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="returnTimeLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>归还时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="returnDateTime">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDialogButtonBox" name="borrowRecordSubmitButton">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="entryPage">
      <attribute name="title">
       <string>录入</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="9">
        <widget class="QLabel" name="outputFileTitleUseableLabel">
         <property name="text">
          <string>为空</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="contentLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>正文：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="7">
        <widget class="QLineEdit" name="fileTitle">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="8" colspan="2">
        <widget class="QGroupBox" name="secLevelRadioButtonBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QRadioButton" name="sec1">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>秘密</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="sec0">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>绝密</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QRadioButton" name="sec3">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>普通</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="sec2">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>机密</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QDateTimeEdit" name="fileSaveTime">
         <property name="minimumSize">
          <size>
           <width>176</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>176</width>
           <height>27</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>2022</year>
           <month>12</month>
           <day>31</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy/M/d hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="10">
        <widget class="QTextEdit" name="fileContent">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="fileTitleLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>档案名称：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="secLevelLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保密等级：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDateTimeEdit" name="fileLoadTime">
         <property name="minimumSize">
          <size>
           <width>176</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>176</width>
           <height>27</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>2022</year>
           <month>11</month>
           <day>30</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy/M/d hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="fileLoadTimeLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>开始时间：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="fileSaveTimeLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>结束时间：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="9">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>945</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="8">
        <widget class="QDialogButtonBox" name="saveButtonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="queryPage">
      <attribute name="title">
       <string>查询</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTabWidget" name="queryTab">
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>借阅记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0" colspan="3">
            <widget class="QTableView" name="queryBorrowRecordTable"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>注：双击已批准记录的文件名方格来查看文件</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="returnSelectRecordButton">
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>提前归还选中</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="refushBorrowRecordButton">
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>刷新借阅记录</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controlPage">
      <attribute name="title">
       <string>管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QToolBox" name="controlBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="fileControl">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>916</width>
            <height>422</height>
           </rect>
          </property>
          <attribute name="label">
           <string>文件数据管理</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QPushButton" name="refreshFile">
             <property name="text">
              <string>刷新文件</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="checkOverdueFile">
             <property name="text">
              <string>检查过期文件</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="deleteSelectedFile">
             <property name="text">
              <string>删除选中文件</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QTableView" name="fileTable"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="manageControl">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>916</width>
            <height>422</height>
           </rect>
          </property>
          <attribute name="label">
           <string>用户权限管理</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="2">
            <widget class="QPushButton" name="deleteUser">
             <property name="text">
              <string>删除用户</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QTableView" name="userTable"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="upLevel">
             <property name="text">
              <string>提升权限</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="downLevel">
             <property name="text">
              <string>降低权限</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="borrowRecordControl">
          <attribute name="label">
           <string>借阅记录管理</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" colspan="2">
            <widget class="QTableView" name="borrowRecordTable"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="acceptRecordButton">
             <property name="text">
              <string>批准选中</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="refuseRecordButton">
             <property name="text">
              <string>拒绝选中</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>fileContent</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onFileContentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onFileTitleChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveTime</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onSaveTimeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileLoadTime</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onLoadTimeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabBarClicked(int)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onTabWidgetChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onSaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onCancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>controlBox</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onControlPageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onRefreshFileButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkOverdueFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onCheckOverdueFileButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteSelectedFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onDeleteSelectedFileButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upLevel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onUpLevelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downLevel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onDownLevelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteUser</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onDeleteUserButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>792</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptRecordButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onAcceptRecordButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refuseRecordButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onRefuseRecordButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borrowFileTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onBorrowFileTitleChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returnDateTime</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onReturnTimeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borrowRecordSubmitButton</sender>
   <signal>accepted()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onSubmitBorrowRecordButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borrowRecordSubmitButton</sender>
   <signal>rejected()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onCancelBorrowRecordButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>827</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returnSelectRecordButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onReturnSelectRecordButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBorrowRecordTable</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onGetContentDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>957</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refushBorrowRecordButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onRefreshBorrowRecordButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>931</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryTab</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onQueryTabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFileContentChanged()</slot>
  <slot>onFileTitleChanged(QString)</slot>
  <slot>onSaveTimeChanged()</slot>
  <slot>onLoadTimeChanged()</slot>
  <slot>onTabWidgetChanged(int)</slot>
  <slot>onSaveButtonClicked()</slot>
  <slot>onCancelButtonClicked()</slot>
  <slot>onControlPageChanged(int)</slot>
  <slot>onRefreshFileButtonClicked()</slot>
  <slot>onCheckOverdueFileButtonClicked()</slot>
  <slot>onDeleteSelectedFileButtonClicked()</slot>
  <slot>onUpLevelButtonClicked()</slot>
  <slot>onDownLevelButtonClicked()</slot>
  <slot>onDeleteUserButtonClicked()</slot>
  <slot>onAcceptRecordButtonClicked()</slot>
  <slot>onRefuseRecordButtonClicked()</slot>
  <slot>onBorrowFileTitleChanged(QString)</slot>
  <slot>onReturnTimeChanged()</slot>
  <slot>onSubmitBorrowRecordButtonClicked()</slot>
  <slot>onCancelBorrowRecordButtonClicked()</slot>
  <slot>onRefreshBorrowRecordButtonClicked()</slot>
  <slot>onGetContentDoubleClicked(QModelIndex)</slot>
  <slot>onReturnSelectRecordButtonClicked()</slot>
  <slot>onQueryTabChanged(int)</slot>
 </slots>
</ui>
